api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -388,39 +391,71 @@ public void sendFileLengthsEgressRemediationEmail(DbUser dbUser, EgressRemediati
   }
 
   @Override
-  public void sendPublishWorkspaceByAdminEmail(
+  public void sendWorkspaceIsFeaturedByAdminEmail(
       DbWorkspace workspace, List<DbUser> owners, FeaturedWorkspaceCategory publishCategory)
       throws MessagingException {
-    sendPublishUnpublishWorkspaceByAdminEmail(
+    sendFeaturedWorkspaceByAdminEmail(
         workspace, owners, featuredWorkspaceCategoryAsDisplayString(publishCategory), true);
   }
 
   @Override
-  public void sendUnpublishWorkspaceByAdminEmail(DbWorkspace workspace, List<DbUser> owners)
+  public void sendWorkspaceIsNotFeaturedByAdminEmail(DbWorkspace workspace, List<DbUser> owners)
       throws MessagingException {
-    sendPublishUnpublishWorkspaceByAdminEmail(workspace, owners, "", false);
+    sendFeaturedWorkspaceByAdminEmail(workspace, owners, "", false);
   }
 
-  private void sendPublishUnpublishWorkspaceByAdminEmail(
-      DbWorkspace workspace, List<DbUser> owners, String cateogryIfAny, boolean publish)
+  @Override
+  public void sendWorkspaceIsCommunityWorkspaceEmail(DbWorkspace workspace, List<DbUser> owners)
       throws MessagingException {
-    String actionType = publish ? "published" : "unpublished";
+    sendMarkAsFeaturedWorkspaceEmail(
+        workspace, owners, FeaturedWorkspaceCategory.COMMUNITY.toString(), true, false);
+  }
 
-    final String ownersForLogging =
-        owners.stream().map(this::userForLogging).collect(Collectors.joining(", "));
+  private void sendFeaturedWorkspaceByAdminEmail(
+      DbWorkspace workspace, List<DbUser> owners, String categoryIfAny, boolean featured)
+      throws MessagingException {
+    sendMarkAsFeaturedWorkspaceEmail(workspace, owners, categoryIfAny, featured, true);
+  }
 
+  private void sendMarkAsFeaturedWorkspaceEmail(
+      DbWorkspace workspace,
+      List<DbUser> owners,
+      String category,
+      boolean featured,
+      boolean isAdmin)
+      throws MessagingException {
+
+    String actionType = featured ? "published" : "unpublished";
+    String ownersForLogging =
+        owners.stream().map(this::userForLogging).collect(Collectors.joining(", "));
     String supportEmail = workbenchConfigProvider.get().mandrill.fromEmail;
 
+    String subject = "Your AoU Researcher Workbench workspace has been " + actionType;
+    String message =
+        isAdmin
+            ? String.format(
+                "%s workspace by admin email for workspace '%s' (%s) sent to owners %s",
+                actionType,
+                workspace.getName(),
+                workspace.getWorkspaceNamespace(),
+                ownersForLogging)
+            : String.format(
+                "Workspace %s (%s) has been published by owner, mail sent to All workspace owners %s",
+                workspace.getName(), workspace.getWorkspaceNamespace(), ownersForLogging);
+
+    String templateResource =
+        featured
+            ? (isAdmin ? FEATURED_WORKSPACE_ADMIN_RESOURCE : FEATURED_WORKSPACE_BY_OWNER_RESOURCE)
```

Comment:
this amount of branching makes me wonder if there's a different way to structure this